sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldUseBaseTypes.cs

```diff
@@ -605,4 +606,16 @@ public void MethodB(B b)
             b.Foo();
         }
     }
+}
+
+namespace Test_22
+{
+    public class Foo
+    {
+        // New test case - code doesn't compile but was making analyzer crash
+        private void Foo(IList<int> a, IList<string> a)
```

Comment:
Let's test the case with no parameter names at all:
`private void Foo(IList<int>, IList<string>)`